ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4760,4 +4760,35 @@ HWRWISOVPM:
   - heating_return_water_isolation_valve_percentage_command
   - heating_return_water_isolation_valve_percentage_sensor
   implements:
-  - MONITORING
\ No newline at end of file
+  - MONITORING
+
+COCDSP:
+  description: "Dual setpoint CO concentration control."
+  is_abstract: true
+  uses:
+  - low_zone_air_co_concentration_setpoint
```

Comment:
can we please use `low_limit` instead of `low`.

Reply:
Done